icegatherer_test.go

```diff
@@ -114,92 +111,3 @@ func TestICEGather_LocalCandidateOrder(t *testing.T) {
 
 	assert.NoError(t, gatherer.Close())
 }
-
-func TestNewICEGatherer_NAT1To1IP(t *testing.T) {
```

Comment:
These tests just assert that values are properly copied from `SettingEngine` -> `ICEGatherer`. I don't think we need them anymore with the indirection removed!